src/cdk/commands/renderGraphCommandPalette.ts

```diff
@@ -0,0 +1,36 @@
+/*!
+ * Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as vscode from 'vscode'
+
+import { PreviewStateMachineCDKWizard } from '../wizards/previewStateMachineCDKWizard'
+import { AslVisualizationCDKManager } from './aslVisualizationCDKManager'
+import { getLogger, Logger } from '../../shared/logger'
+import { renderStateMachineGraphCommand } from './renderStateMachineGraph'
+
+export async function previewCDKStateMachineFromCommandPalette(
+    globalStorage: vscode.Memento,
+    aslVisualizationCDKManager: AslVisualizationCDKManager
+) {
+    const logger: Logger = getLogger()
+
+    const wizardResponse = await new PreviewStateMachineCDKWizard().run()
```

Comment:
this is the only difference compared to `renderStateMachineGraphCommand()`. 36 lines of code is a lot of code to read for a 1-line difference. Can `renderStateMachineGraphCommand()` be parameterized ?